runtime/oti/j9javaaccessflags.h

```diff
@@ -107,6 +107,9 @@
 #define J9AccSealed 0x200
 #define J9AccRecord 0x400
 #define J9AccClassAnonClass 0x800
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+#define J9AccClassIsInjectedInvoker 0x1000
```

Comment:
please also update the map comment for `ROMClassBuilder::computeExtraModifiers` to correctly represent the bit as being used

Reply:
That was what I was forgetting. Thanks for reminding me.